CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
-# dbt_zendesk v0.17.0
+# dbt_zendesk v0.19.0
```

Comment:
Let's make this a pre-release
```suggestion
# dbt_zendesk v0.19.0-a1
```

Reply:
Can you also make the same update in the relevant sections of the documentation.